package.json

```diff
@@ -79,16 +81,45 @@
         }
     },
     "peerDependencies": {
+        "@guardian/consent-management-platform": "^6.6.0",
         "@guardian/src-foundations": "^2.4.0",
         "@guardian/src-icons": "^2.4.0",
         "emotion": "^10.0.27",
         "react": "^16.13.1",
         "react-dom": "^16.13.1"
     },
     "jest": {
+        "preset": "ts-jest/presets/js-with-babel",
```

Comment:
We can revisit this but I've copied the jest config from DCR here to fix errors that I was seeing after adding CMP